### PR TITLE
INSTA-3500: Documenting Synthetic SSLCertificate Tests

### PR DESCRIPTION
**Pull request by andreachiavegatto**

### CODE REVIEW

File: spec/descriptions/tagSyntheticSettings.md [hunk 1]
```diff
@@ -88,6 +88,10 @@ The API endpoints of this group can be used to manage Synthetic Locations, Synth
           - **browser** The type of the browser: chrome or firefox, chrome by default.
           - **recordVideo** A boolean type, false by default.
           - **syntheticType** Its value is WebpageScript. It is required.
+      - **SSLCertificateConfiguration** has the following properties:
+          - **hostname** The hostname of the SSL enabled website.
+          - **port** The SSL port, set to 443 by default.
+          - **daysRemainingCheck** The number of days to use on the validation check. The test will fail when the certificate validity has less this number of days remaining until expiration.
```
Comment (ediazgallego) [inline spec/descriptions/tagSyntheticSettings.md:94]:
```suggestion
          - **daysRemainingCheck** The number of days to use on the validation check. The test will fail when the certificate validity has less than this number of days remaining until expiration.
```

**Review by ediazgallego — CHANGES_REQUESTED** [review]
Looks good overall, small correction for the daysRemainingCheck text

File: spec/descriptions/tagSyntheticSettings.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ The API endpoints of this group can be used to manage Synthetic Locations, Synth
         - If timeout value in test configuration is not provided, the default value is **1m** for HTTPAction and HTTPScript tests. 
           BrowserScript, WebpageAction, and WebpageScript tests use the smaller one of `maxTimeout` and `testFrequency` as the actual timeout value.
     - **XXXConfiguration** The configuration corresponding to the syntheticType. Configuration types are HTTPActionConfiguration, HTTPScriptConfiguration,
-      BrowserScriptConfiguration (Beta), WebpageActionConfiguration (Beta), and WebpageScriptConfiguration (Beta). 
+      BrowserScriptConfiguration, WebpageActionConfiguration, WebpageScriptConfiguration, and SSLCertificateConfiguration (Beta).
```
Comment (carolzhang65) [inline spec/descriptions/tagSyntheticSettings.md:46]:
```suggestion
      BrowserScriptConfiguration, WebpageActionConfiguration, WebpageScriptConfiguration, and SSLCertificateConfiguration. 
```

Comment (carolzhang65) [reply spec/descriptions/tagSyntheticSettings.md:46]: A similar sentence about Synthetic types is also in Location properties part at this [line](https://github.com/instana/openapi/blob/b8bef1f90539cc2625845191d9839b770901f588/spec/descriptions/tagSyntheticSettings.md?plain=1#L11). It needs to be updated too.

**Review by ediazgallego — APPROVED** [review]
:1st_place_medal:  looks good now, thanks!

**Review by carolzhang65 — COMMENTED** [review]
"testFrequency" range needs to be updated, please state that for SSLCertificate type, the default is every 24 hours. 

In Synthetic result file, I think we'd better to give an example about how to get SSLCertificate custom_metrics data in this [section](https://github.com/instana/openapi/blob/b8bef1f90539cc2625845191d9839b770901f588/spec/descriptions/tagSyntheticResults.md?plain=1#L123).

File: spec/descriptions/tagSyntheticSettings.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ The API endpoints of this group can be used to manage Synthetic Locations, Synth
 - **playbackCapability** The playback capabilities provided by this location resource.
   The playbackCapability object has the following properties: 
   - **syntheticType** Different types of synthetic tests that can be executed at this location. 
-    Possible values are HTTPAction, HTTPScript, BrowserScript (Beta), WebpageAction (Beta), and WebpageScript (Beta). 
+    Possible values are HTTPAction, HTTPScript, BrowserScript, WebpageAction, WebpageScript, and SSLCetificate.
```
Comment (KarenGdaniec) [inline spec/descriptions/tagSyntheticSettings.md:11]:
```suggestion
    Possible values are HTTPAction, HTTPScript, BrowserScript, WebpageAction, WebpageScript, and SSLCertificate. 
```

**Review by KarenGdaniec — COMMENTED** [review]
Agree with Carol's comments, good catches.

**Comment by andreachiavegatto** [comment]
> In Synthetic result file, I think we'd better to give an example about how to get SSLCertificate custom_metrics data in this section

@carolzhang65 There are no examples of the metric returned, just the payload used to query them. 
Should we add one for the SSLCertificate custom metrics?

**Review by carolzhang65 — APPROVED** [review]
The changes look good to me. Thanks.

**Comment by carolzhang65** [comment]
> 
> @carolzhang65 There are no examples of the metric returned, just the payload used to query them. Should we add one for the SSLCertificate custom metrics?

No, a sample request payload is good enough. Thanks.

**Pull request merged**